optimize/backend/src/main/java/io/camunda/optimize/TomcatConfig.java

```diff
@@ -0,0 +1,220 @@
+      "application/json," + "text/html," + "application/x-font-ttf," + "image/svg+xml";
+  private static final String PROTOCOL = "http/1.1";
+  private static final String LOGIN_ENDPOINT = "/login";
+  private static final String METRICS_ENDPOINT = "/metrics";
+  private static final String URL_BASE = "/#";
+
+  public static final String EXTERNAL_SUB_PATH = "/external";
+  public static final String ALLOWED_URL_EXTENSION =
+      String.join(
+          "|",
+          new String[] {
+            "/",
+            URL_BASE,
+            LOGIN_ENDPOINT,
+            METRICS_ENDPOINT,
+            CCSaasAuth0WebSecurityConfig.OAUTH_AUTH_ENDPOINT,
+            CCSaasAuth0WebSecurityConfig.OAUTH_REDIRECT_ENDPOINT,
+            CCSaasAuth0WebSecurityConfig.AUTH0_JWKS_ENDPOINT,
+            CCSaasAuth0WebSecurityConfig.AUTH0_AUTH_ENDPOINT,
+            CCSaasAuth0WebSecurityConfig.AUTH0_TOKEN_ENDPOINT,
+            CCSaasAuth0WebSecurityConfig.AUTH0_USERINFO_ENDPOINT,
+            HealthRestService.READYZ_PATH,
+            LocalizationRestService.LOCALIZATION_PATH,
+            TomcatConfig.EXTERNAL_SUB_PATH,
+            OptimizeResourceConstants.REST_API_PATH,
+            OptimizeResourceConstants.STATIC_RESOURCE_PATH,
+            OptimizeResourceConstants.ACTUATOR_ENDPOINT,
+            PanelNotificationConstants.SEND_NOTIFICATION_TO_ALL_ORG_USERS_ENDPOINT,
+            RestConstants.BACKUP_ENDPOINT,
+            UIConfigurationRestService.UI_CONFIGURATION_PATH
+          });
+
+  private static final String HTTP11_NIO_PROTOCOL = "org.apache.coyote.http11.Http11Nio2Protocol";
+
+  @Autowired private ConfigurationService configurationService;
+  @Autowired private Environment environment;
+
+  private void setSendServerVersion(Connector connector, boolean toggle) {
+    connector.setXpoweredBy(toggle);
+  }
+
+  @Bean
+  public FilterRegistrationBean<ExternalApiRewriteFilter> externalApiRewriteFilter() {
+    FilterRegistrationBean<ExternalApiRewriteFilter> filterRegistrationBean =
+        new FilterRegistrationBean<>();
+    filterRegistrationBean.setFilter(new ExternalApiRewriteFilter());
+    filterRegistrationBean.addUrlPatterns("/*");
+    return filterRegistrationBean;
+  }
+
+  @Bean
+  public ServletContextInitializer externalResourcesHandler() {
+    return new ServletContextInitializer() {
+      @Override
+      public void onStartup(ServletContext servletContext) throws ServletException {
+        LOG.debug("Registering bean externalResourcesHandler...");
+        URL webappURL = getClass().getClassLoader().getResource("webapp");
+        if (webappURL == null) {
+          LOG.debug("Static content directory 'webapp' not found. No bean will be registered.");
+          return;
+        }
+
+        String webappPath = webappURL.toExternalForm().replaceFirst("file:", "");
+        ServletRegistration.Dynamic webappServlet =
+            servletContext.addServlet("external-home", ExternalHomeServlet.class);
```

Comment:
To do this, I had to implement a custom servlet that serves the static files. I didn't find a more immediate or automatic way to do it with Tomcat. If you know how to do it better, please feel free to let me know